File: official/docs/python/v8/addresses/verify.py
import easypost
import os

client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))

address = client.address.create(
    street1="417 MONTGOMERY ST",
    street2="FLOOR 5",
    city="San Francisco",
    state="CA",
    zip="94104",
    country="US",
    company="EasyPost",
    phone="415-123-4567",
)

verified_address = client.address.verify(address.id)

print(verified_address)

File: official/docs/node/v5/addresses/verify.js
const Easypost = require('@easypost/api');

const api = new Easypost('EASYPOST_API_KEY');

const address = new api.Address({
  street1: '417 montgomery streat',
  city: 'SAN FRANCISCO',
  state: 'CA',
  zip: '94104',
  country: 'US',
  company: 'EasyPost',
  phone: '415-123-4567',
});

address.save().then((createdAddress) => {
  createdAddress.verifyAddress().then(console.log);
});
